pipelines/install/04-install_proxy.yaml

```
@@ -16,6 +16,7 @@
       - "--foreman-proxy-oauth-consumer-key {{ oauth_consumer_key.stdout }}"
       - "--foreman-proxy-oauth-consumer-secret {{ oauth_consumer_secret.stdout }}"
       - "--foreman-proxy-content-parent-fqdn {{ server_fqdn.stdout }}"
+      - "{{ '' if (pipeline_version == 'nightly' or 2.3 is version(pipeline_version, '>')) else ('--foreman-proxy-content-parent-fqdn %s' % server_fqdn.stdout) }}"
```

Just a question: does the following work as well?
```suggestion
      - "{{ ('--foreman-proxy-content-parent-fqdn %s' % server_fqdn.stdout) if (2.3 is version(pipeline_version, '<=')) else '' }}"
```

---

are you talking about the re-ordering, or droping of the "is nightly" drop?

anyways, no it will not: `version()` can't compare `nightly` to `2.3`, so we never should pass `nightly` to it.

---

It was the latter part.